### ISSUE TITLE
Kodi crash when changing source (live tv / video) > 5.0.1

### ISSUE THREAD

**Issue by posixx** [OpenELEC/OpenELEC.tv#3971]
Everytime i want to start another video source within Kodi (except for the first selected souce after boot), the kodi process terminates. Kodi is starting up automatically and i can successfully start the source then. But after changing it again kodi again crashes. 

This happens for ALL v5 releases EXCEPT 5.0.0 which is rock solid. I'm using Haswell CPU with ASUS Z87 mobo.

The only thing i see in the openelec hardware log:

[   73.410587] traps: kodi.bin[732] general protection ip:7f8d1aa8abf0 sp:7fff042d9238 error:0 in libpthread-2.20.so[7f8d1aa79000+17000]

If i can do anything generating correct logs / traces let me know.

**Comment by fritsch** [comment]
We basically need everything the contributing link told you: https://github.com/OpenELEC/OpenELEC.tv/blob/master/CONTRIBUTING.md

Please do a memtest on your system.

**Comment by posixx** [comment]
Fritsch,

As said i am running 5.0.0 (and before openelec i was running the custom sse4 builds) without any problems. As soon as i try another release after 5.0.0 i get the crashes. So it's not a memory issue.

You can download the log-zip here: http://support.yodacloud.net/downloads/log-2015-03-02-13.57.08.zip

**Comment by posixx** [comment]
And again with Log debug enabled within Kodi:  http://support.yodacloud.net/downloads/log-2015-03-02-14.46.02.zip

**Comment by fritsch** [comment]
Thanks for the logfiles.

I can only guess: 
- Does it also happen when you disable PVR completely?
- Does it happen if you only remove pulsar?
- Does the setting: Do not save EPG in database enabled change anything?

I think we can't solve it without a gdb backtrace and a devel build.

**Comment by posixx** [comment]
Hello fritsch,

Sorry it took a while to have some time to try your suggestions. I have upgraded to 5.0.5 and disabled pulsar and LiveTV and addons involved. After boot i started a mkv file, immediately after stopping the stream Kodi crashed: http://support.yodacloud.net/downloads/log-2015-03-05-11.18.05.zip

Back to 5.0.0 for now; if you need me to have a devel/gdb build tested let me know, i would be happy testing for you. BTW i have an Intel NUC with Haswell, same settings same addons and that is running 5.0.5 without any problems.

**Comment by fritsch** [comment]
Good: Can you try this build, please: https://dl.dropboxusercontent.com/u/55728161/OpenELEC-Generic.x86_64-devel-20150304081919-r20097-g92ed225.tar

**Comment by fritsch** [comment]
```
[   53.458612] traps: kodi.bin[730] general protection ip:7f40770d7bf0 sp:7fffa5d655e8 error:0 in libpthread-2.20.so[7f40770c6000+17000]
```

is exactly what: https://github.com/OpenELEC/OpenELEC.tv/commit/2dbcf229015da09b2f82b059baa8b30444d8b99b should fix

**Comment by posixx** [comment]
Yes! Fixed in this dev build. Great! So i guess this will be available with 5.0.6 release?

Thanks again fritsch

**Comment by fritsch** [comment]
Thanks for reporting back - yes I pushed it into OE 5.0.x branch. That was a bug where I never had expected it ... in libva ...

**Comment by posixx** [comment]
Great. Many thanks for support. Closing now..

**Comment by fritsch** [comment]
Let's keep it open until 5.0.6 is released.

**Comment by tunf** [comment]
5.0.6 is out, so close this one?

**Comment by posixx** [comment]
Yes confirming solved in 5.0.6!
